lib/combine_pdf/parser.rb

```diff
@@ -58,6 +59,7 @@ def initialize (string)
 			@strings_dictionary = {} # all strings are one string
 			@version = nil
 			@scanner = nil
+			@allow_optional_content = options[:allow_optional_content] || false
```

Comment:
Is the `false` value necessary? the default missing key value is `nil`, which evaluates the same...